keymaps/git.cson

```diff
@@ -3,6 +3,19 @@
   'ctrl-shift-8': 'github:toggle-github-tab'
   'ctrl-9': 'github:toggle-git-tab-focus'
   'ctrl-shift-9': 'github:toggle-git-tab'
+  'alt-g enter': 'github:commit'
+  'alt-g f': 'github:fetch'
+  'alt-g F': 'github:pull'
```

Comment:
I'd love it if we could do alt-g up-arrow and alt-g down-arrow but I'm pretty sure we don't support that kind of keybinding today.